docs/_documentations/mdt-che-installinfo.md

```diff
@@ -1,62 +1,47 @@
+   - For Network File System (NFS), set 777 permissions for the exported folders and ownership of `nobody:nogroup`.
+   - Because Codewind uses RWX volumes to provide persistent storage, use NFS for storage on OpenShift 4.
 2. Ensure that the cluster can pull images from `docker.io/eclipse` and `quay.io/eclipse`.
-   - Both Eclipse Che and Eclipse Codewind host their Docker images at those locations. Ensure that your cluster can pull from that registry and does not have `ImagePullPolicies` blocking it from accessing Docker Hub.
-3. Set up the ClusterRole for Codewind. Run `kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.8.0/setup/install_che/codewind-clusterrole.yaml`
-
-# Setting up Che
-
-## Installing Che with chectl
-
-The fastest way to install Eclipse Che for use with Codewind is to use the `chectl` CLI. For instructions on installing the `chectl` CLI tool, see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/).
+   - Both Eclipse Che and Eclipse Codewind host Docker images at those locations. Your cluster needs to pull from that Docker registry, and `ImagePullPolicies` cannot block the cluster from accessing Docker Hub.
+3. Set up the ClusterRole for Codewind by running the `kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.8.0/setup/install_che/codewind-clusterrole.yaml` command.
 
-Complete the following steps after you install `chectl`:
+### Installing Che with chectl
+The fastest way to install Eclipse Che for Codewind is to use the `chectl` CLI. To install the `chectl` CLI tool, see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/).
 
-1. Download the [codewind-checluster.yaml](https://github.com/eclipse/codewind-che-plugin/blob/0.8.0/setup/install_che/che-operator/codewind-checluster.yaml) file to your machine.
+After you install `chectl`, complete the following steps:
+1. Download the [codewind-checluster.yaml](https://github.com/eclipse/codewind-che-plugin/blob/0.8.0/setup/install_che/che-operator/codewind-checluster.yaml) file.
     - You can modify this file, but leave the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and the `spec.storage.preCreateSubPaths` field set to `true`.
-2. If you're installing on a Kubernetes platform other than OpenShift, determine your Ingress domain. If you're unsure of your Ingress domain, ask your cluster administrator.
-    - Set the `spec.server.ingressDomain` field in the Che `.yaml` to the Ingress domain.
+2. If you're installing Che on a Kubernetes platform other than OpenShift, determine your Ingress domain. If you're unsure of your Ingress domain, ask your cluster administrator. Set the `spec.server.ingressDomain` field in the Che `.yaml` file to the Ingress domain.
```

Comment:
I prefer to have the `Set the spec.server.ingressDomain field in the Che .yaml file to the Ingress domain.` on a separate line because users are more likely to miss the instruction if it's on the same line

Reply:
I'll make this info on a separate line by giving it its own step.